### ISSUE TITLE
Load Balancer / Scaling Group association fails on first run

### ISSUE THREAD

**Issue by thomshutt** [terraform-providers/terraform-provider-alicloud#1436]
### Terraform Version
Terraform v0.12.3
provider.alicloud: version = "~> 1.52"

### Affected Resource(s)
- slb

### Terraform Configuration Files
```hcl
variable "access_key" {}
variable "secret_key" {}

provider "alicloud" {
  access_key = "${var.access_key}"
  secret_key = "${var.secret_key}"
  region = "ap-southeast-1"
}

data "alicloud_zones" "zones_ds" {
  available_instance_type = "ecs.t5-lc2m1.nano"
  available_disk_category  = "cloud_ssd"
}

resource "alicloud_vpc" "vpc" {
  name = "vpc"
  cidr_block = "10.0.0.0/16"
}

resource "alicloud_vswitch" "vswitch-1" {
  name = "vswitch-1"
  vpc_id            = "${alicloud_vpc.vpc.id}"
  cidr_block        = "10.0.0.0/18"
  availability_zone = "${data.alicloud_zones.zones_ds.zones.0.id}"
}

resource "alicloud_ess_scaling_group" "ess-scaling-group" {
  min_size = 2
  max_size = 2
  scaling_group_name = "scalinggroup"
  removal_policies = ["OldestInstance", "NewestInstance"]
  vswitch_ids = ["${alicloud_vswitch.vswitch-1.id}"]
  loadbalancer_ids = ["${alicloud_slb.slb.id}"]
  multi_az_policy = "BALANCE"
}


resource "alicloud_slb" "slb" {
  name                 = "terraform-slb"
  internet             = false
  bandwidth            = 2
  vswitch_id = "${alicloud_vswitch.vswitch-1.id}"
}

resource "alicloud_slb_listener" "tcp_http" {
  load_balancer_id = "${alicloud_slb.slb.id}"
  backend_port = "22"
  frontend_port = "22"
  protocol = "tcp"
  bandwidth = "10"
  health_check_type = "tcp"
}

resource "alicloud_eip" "eip" {
  name = "eip"
  internet_charge_type = "PayByTraffic"
}

resource "alicloud_eip_association" "eip_load_balancer_association" {
  allocation_id = "${alicloud_eip.eip.id}"
  instance_id   = "${alicloud_slb.slb.id}"
}
```

### Expected Behavior

An SLB is created and associated with an EIP.

### Actual Behavior

```
Error: [ERROR] terraform-provider-alicloud/alicloud/resource_alicloud_ess_scalinggroup.go:115: Resource alicloud_ess_scalinggroup CreateScalingGroup Failed!!! [SDK alibaba-cloud-sdk-go ERROR]:
SDK.ServerError
ErrorCode: IncorrectLoadBalancerStatus
Recommend: 
RequestId: 418A3514-5FCD-44FB-83EE-9C8638D44787
Message: The current status of the load balancer "lb-dj1fwbe94il6nu434b0j2" does not support this action.
```

Re-running `terraform apply` results in successful creation.

### Steps to Reproduce
`terraform apply`

### References

Seems like a regression of https://github.com/alibaba/terraform-provider/issues/314

**Comment by xiaozhu36** [comment]
HI @thomshutt It seems like the error message does not match your issue title. Can you clear it?

**Comment by thomshutt** [comment]
Sorry, updated - too many acronyms!

**Comment by TalentNing** [comment]
@thomshutt loadBalancer need to be "active" status before attached into scalingGroup, slb may need a few seconds before entrance into active status.

**Comment by thomshutt** [comment]
@TalentNing Is there recommended solution here (i.e to "wait" for the slb) in Terraform?

**Comment by TalentNing** [comment]
@thomshutt I have review the code for alicloud_slb, it already waits for entrancing 'active' status when create.
Does this error always hanppen or occasionally ？

**Comment by thomshutt** [comment]
@TalentNing Always. You should be able to recreate the issue with the sample Terraform Config on this ticket.

**Comment by GabrielZabielski** [comment]
Same issue, code below returns the same error, however, not all the times - sometimes it's right.
`
module "slb-listener" { ... }

resource "null_resource" "delay" {
  provisioner "local-exec" {
    command = "sleep 20"
  }
 // with triggers block is the same result... 
  depends_on = [module.server_load_balancer.slb_id] // i tryed to add there listener too.
}

resource "alicloud_ess_scaling_group" "that" {
  ...
  depends_on = [null_resource.delay]
`

Apply logs clip:
`
module.beijing.module.server_load_balancer.alicloud_slb.this: Creation complete after 7s [id=lb-xxxxxxxx]
module.beijing.null_resource.delay: Creating...
module.beijing.null_resource.delay: Provisioning with 'local-exec'...
module.beijing.module.slb-listener.alicloud_slb_listener.this: Creating...
module.beijing.module.auto_scaling.alicloud_ess_scaling_group.this[0]: Creating...
module.beijing.null_resource.delay (local-exec): Executing: ["/bin/sh" "-c" "sleep 20"]
`

Apply before this faile ends without errors.

**Comment by TalentNing** [comment]
@thomshutt @GabrielZabielski  I have tried with the same TF config. It seems slb may sometime in 'inactive' status, although it has been checked when creating slb。I will add some spare time to wait for 'active' status of attached slb when creating scaling group. It may help a lot.

**Comment by GabrielZabielski** [comment]
.   (Tactical reminding pointer :smile: )

**Comment by xiaozhu36** [comment]
Fixed by 1.54.0

**Comment by GabrielZabielski** [comment]
https://github.com/terraform-providers/terraform-provider-alicloud/issues/1628
